src/ex4/package.json

```diff
@@ -15,6 +15,10 @@
     "cors": "^2.8.5",
```

Comment:
you're missing the mysql2 package, I couldn't run it locally until installing it myself